src/app.cpp

```diff
@@ -88,8 +88,40 @@ class simulationManager {
     }
 };
 
+class PythonKappaBarController : public baseController
```

Comment:
Please change o the current style of lowercase followed by camel for class names: `pythonKappaBarController`.

I actually want to change class names to camel case in a future refactor, but for now we can keep the style consistent.